test/jdk/javax/sound/midi/SysexMessage/SendRawSysexMessage.java

```diff
@@ -113,6 +114,16 @@ private static void test(MidiDevice.Info info) throws Exception {
                         (byte) SPECIAL_SYSTEM_EXCLUSIVE}), -1);
                 System.err.println("note off");
                 r.send(new ShortMessage(ShortMessage.NOTE_OFF, 5, 5), -1);
+                System.err.println("sysex part 1 of 3");
+                r.send(new SysexMessage(new byte[]{
+                        (byte) SYSTEM_EXCLUSIVE, 0x7D, 0x01, 0x02}, 4), -1);
+                System.err.println("sysex part 2 of 3");
+                r.send(new SysexMessage(new byte[]{
+                        (byte) SPECIAL_SYSTEM_EXCLUSIVE, 0x03, 0x04}, 3), -1);
+                System.err.println("sysex part 3 of 3");
+                r.send(new SysexMessage(new byte[]{
+                        (byte) SPECIAL_SYSTEM_EXCLUSIVE, 0x05, 0x06, 0x07
+                        (byte) SPECIAL_SYSTEM_EXCLUSIVE}, 4), -1);
```

Comment:
comma is missed?

Reply:
Fixed. I apologize for the stupid mistake.